src/algebra/ordered_group.lean

```diff
@@ -820,6 +820,9 @@ end
 lemma max_sub_min_eq_abs (a b : α) : max a b - min a b = abs (b - a) :=
 by { rw abs_sub_comm, exact max_sub_min_eq_abs' _ _ }
 
+/-
+**Triangle inequality**
```

Comment:
Here and below: docstring comments begin with `/--` (and end with `-/`). It may also be worth specifying the types or saying something brief about the version that's formalized there.
```suggestion
/--
The **triangle inequality** in `linear_ordered_add_comm_group`s.
```